iFixit/checkout-ui-extensions
# EU pro address validation

Closes https://github.com/iFixit/checkout-ui-extensions/issues/41

Blocks the customer from progressing through the checkout if their shipping address country does not match their default address, or if any part of their billing address does not match their default address.

This validation is performed on the payment method page. We could do it on the contact info page too for the shipping address country, but I chose to keep all the validation in one place here.

## QA

If https://github.com/iFixit/ifixit/pull/52446 is not yet merged, you'll need to checkout that branch on cominor and change `useIfixitBaseUrl` to return your subdomain.

Run local dev and preview the `purchase.checkout.payment-method-list.render-after` url.

The "default address" is found on your customer profile in Shopify Admin.

The feature should block your checkout progress with a nice error message if the addresses don't match as described above, and once the addresses match the customer should be able to continue checkout.
